src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx

```diff
@@ -213,26 +210,9 @@ export const CheckoutPayment = ({
     }
   }, [is3DsError, history, trackOfferEvent])
 
-  const addPaymentToCart = useCallback(
-    async (paymentTokenId) => {
-      try {
-        await addPaymentTokenMutation({
-          variables: {
-            id: quoteCartId,
-            paymentTokenId,
-          },
-          refetchQueries: ['QuoteCart'],
-        })
-      } catch (error) {
-        trackOfferEvent({
-          eventName: EventName.SignError,
-          options: { error, errorType: ErrorEventType.PaymentTokenMutation },
-        })
-        console.error('Failed to add Payment Token :', error.message, error)
-      }
-    },
-    [addPaymentTokenMutation, quoteCartId, trackOfferEvent],
-  )
+  const addPaymentToCart = useCallback(async () => {
+    setIsPaymentConnected(true)
+  }, [])
```

Comment:
Don't think this is necessary - you don't need to add `useState` function to the dependency array :)

I think the name of this function is confusing. I would just call it inside the `useEffect`.